tests/view/test_extents.cpp

```diff
@@ -23,6 +24,14 @@ namespace Kokkos::utils::tests::view
 
 constexpr size_t dim_1 = 5, dim_2 = 3, dim_3 = 6, dim_4 = 9;
 
+//! @test Check that @ref view::extents works as expected for a rank-0 @c Kokkos::View.
+TEST(view, extents_rank_0)
+{
+    const Kokkos::View<double, execution_space> view_0("rank-0 view");
+
+    ASSERT_EQ(utils::view::extents(view_0), (Kokkos::Array<size_t, 0>{}));
```

Comment:
OK. May be worth a try to make it a `static_assert`?

Reply:
OK, done.